front/src/components/LoginPanel/comp/RegisterForm.js

```diff
@@ -1,10 +1,21 @@
 import React, { useState } from "react";
+import "./RegisterForm.css";
 import Button from "../../UI/Button/Button";
 import { gql, useMutation } from "@apollo/client";
 
 const REGISTER_USER = gql`
-    mutation register($username: String!, $email: String!, $password: String!) {
-        register(username: $username, email: $email, password: $password) {
+    mutation register(
+        $username: String!
+        $password: String!
+        $email: String!
+        $reason_for_creating_account: String!
+    ) {
+        register(
+            username: $username
+            password: $password
+            email: $email
+            reason_for_creating_account: $reasonForCreatingAccount
+        ) {
```

Comment:
You used reason_for_creating_account and reasonForCreatingAccount, probably should be reason_for_creating_account everywhere. Currently it causes 
![image](https://user-images.githubusercontent.com/44038381/120184683-8c5bef80-c211-11eb-8a9a-2d7296cbdd87.png)